### PR TITLE
feat(ox): qbx native bridge

### PR DESCRIPTION
**Pull request by solareon**
## Description

Needed for ox_inventory bridge. Blocked on https://github.com/Qbox-project/qbx_core/pull/528 and https://github.com/overextended/ox_inventory/pull/1763

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by KostaZx — APPROVED** [review]

**Comment by solareon** [comment]
Don't merge until the other two are merged

**Review by mafewtm — APPROVED** [review]

**Pull request merged**
